test/fastify-mysql.test.js

```diff
@@ -132,74 +131,20 @@ test('should throw without name option and multiple instances', ({ ok, same, pla
   })
 })
 
-test('should create a single query and execute it with the tx() method', ({ error, ok, plan }) => {
+test('should create a single query and execute it with the transaction() method', ({ error, ok, plan }) => {
   plan(2)
 
   const fastify = Fastify()
-  fastify.register(fastifyMysql, options)
-
-  fastify.ready(async (err) => {
-    error(err)
-
-    const queryArray = ['SELECT NOW();']
-
-    const result = await fastify.mysql.tx(queryArray)
-    ok(result.length)
-
-    fastify.close()
-  })
-})
-test('should create a batch of queries and execute it with the tx() method', ({ error, ok, plan }) => {
-  plan(2)
-
-  const fastify = Fastify()
-  fastify.register(fastifyMysql, options)
+  fastify.register(fastifyMysql, { ...options, name: 'first_db' })
 
   fastify.ready(async (err) => {
     error(err)
 
-    const queryArray = ['SELECT NOW();', 'SELECT NOW();', 'SELECT NOW();', 'SELECT NOW();', 'SELECT NOW();', 'SELECT NOW();']
+    const queryArray = ['SELECT NOW()']
```

Comment:
Can you add more queries to the `array`? Just to test the queries execution.

Also, check for the result.

Reply:
Tested the queries, results match.

Tried this new approach, I was having issues mixing asynchronicity, database querying and pure functional style, let me know what you think about